hikari/impl/buckets.py

```diff
@@ -591,17 +620,32 @@ def update_rate_limits(
                 remaining_header,
             )
         else:
-            bucket = RESTBucket(real_bucket_hash, compiled_route)
+            unknown_bucket_hash = _create_unknown_hash(compiled_route)
+
+            if bucket := self.real_hashes_to_buckets.pop(unknown_bucket_hash, None):
```

Comment:
isn't the whole "real hashes" naming scheme now just a lie here

Reply:
We can consider unknown hashes "real hashes". We can also store them in a separate dict if that is better with you. I think it will be harder to manage and having another dict for no reason is a waste of resources